integration_tests/pyproject.toml

```diff
@@ -25,6 +25,7 @@ websockets = "^9.1"
 toml = "^0.10.2"
 pysha3 = "^1.0.2"
 jsonnet = "^0.18.0"
+eth-account = { git = "https://github.com/mmsqe/eth-account.git", branch = "v0.5.8-rc0" }
```

Comment:
can you open a PR to upstream, ideally we should ref the upstream repo, but if it take too long to merge, a fork is alright in test code.

Reply:
seems not back portable, we need move forward with web3 with bunch of updates

Reply:
I see